src/_pytest/logging.py

```diff
@@ -443,17 +450,15 @@ def caplog(request):
     result._finalize()
 
 
-def get_actual_log_level(config, *setting_names):
-    """Return the actual logging level."""
-
+def get_log_level_for_setting(config: Config, *setting_names: str) -> Optional[int]:
```

Comment:
Thx for improving the name of this function!

Reply:
@thisch you are welcome - care to approve? :)